src/Copy/BrandCopy.elm

```diff
@@ -564,7 +564,7 @@ brandCopy key =
                 turned out I had appendicitis."""
 
         StoryTwo2ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a nurse speaking to a woman vices"
```

Comment:
Maybe "Illustration of a nurse speaking to a woman about sex and drinking" or "...speaking to a woman about vices"

Reply:
"Illustration of a nurse speaking to a woman about contraception and drinking" - fits better with the image and the quote, I believe.